src/1-foundations/2-homotopy-type-theory/08-the-unit-type.rzk.md

```diff
@@ -5,3 +5,41 @@ This is a literate Rzk file:
 ```rzk
 #lang rzk-1
 ```
+
+!!! note "Lemma. Any two elements of $\mathbb{1}$ are equal"
+    For any $x,y:\mathbb{1}$, we have $x = y$.
+
+```rzk
+#def units-eq
+    (x y : Unit)
+    : x = y
+    := path-concat Unit x unit y (Unit-uniq x) (path-sym Unit y unit (Unit-uniq y))
```

Comment:
It should be noted that `refl` also works here, since the uniqueness principle is actually built into Rzk for `Unit` and (dependent) pairs (Σ-types). See [`uniq-prod'`](https://fizruk.github.io/bmstu-rzk-demo-2023/day-1/1-dependent-types.rzk/#define:uniq-prod') and [`uniq-Unit'`](https://fizruk.github.io/bmstu-rzk-demo-2023/day-1/1-dependent-types.rzk/#define:uniq-Unit') in the recent Rzk demo at https://fizruk.github.io/bmstu-rzk-demo-2023